src/handlers/console.ts

```diff
@@ -47,6 +47,7 @@ export class ConsoleHandler implements IDisposable {
     this._cellMap.values().forEach(handler => handler.dispose());
     this._cellMap.dispose();
     Signal.clearData(this);
+    this.consolePanel.node.removeAttribute('data-jp-debugger');
```

Comment:
It looks a bit odd for this attribute to be set at the plugin level, and then removed here on dispose in the handler.

Maybe adding and removing the attribute should happen at the same place?